### DISCUSSION TITLE
Climate card simplified

### DISCUSSION THREAD

**Discussion by narfotic** [piitaya/lovelace-mushroom#719] [Ideas]
I would love for the climate card to do the following:

**A.** Let me choose the default HVAC-mode the front icon toggle switches. Now it selects heating no matter what.

**OR B.** IF only one HVAC mode is selected AND temperature control is selected
Rather than show the 'go to HVAC-selector'-toggle: show the one HVAC mode. And have that toggle to off when pressed when ON.

This would allow me to:
- control everything from my card without having to go into any submenu
- just show the ones with cooling in summer and hide these and show heating ones with a toggle-helper in winter.

**Comment by spacekiek** [comment]
Hi, I'm having the same issue ... the front icon always turns **on** my AC _AND_ selects Heating mode ?
